backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/Lesson/FiltersRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace EasySpeak.Core.Common.DTO.Lesson;
+
+public class FiltersRequest
+{
+    [JsonProperty("languageLevels", Required = Required.Default)]
```

Comment:
In name of property is the same, you don't need to add JsonProperty attribute

Reply:
fixed